k2/csrc/ragged_ops.cu

```diff
@@ -1249,14 +1250,62 @@ std::vector<RaggedShape> GetPrefixes(RaggedShape &src,
 
 Ragged<int32_t> AddSuffixToRagged(Ragged<int32_t> &src,
                                   const Array1<int32_t> &suffix) {
-  K2_LOG(FATAL) << "Not Implemented!";
-  return Ragged<int32_t>();
+  NVTX_RANGE(K2_FUNC);
+  K2_CHECK_EQ(suffix.Dim(), src.TotSize(src.NumAxes() - 2));
```

Comment:
The last axis is `NumAxes() - 1`. 

I think there is a typo in the documentation of the interface.